Extension/src/LanguageServer/client.ts

```diff
@@ -382,6 +396,16 @@ class DefaultClient implements Client {
         }
     }
 
+    public onDidChangeVisibleTextEditors(editors: vscode.TextEditor[]): void {
+        //Apply text decorations to inactive regions
+        for (let e of editors) {
+            let valuePair: decorationRangesPair = this.inactiveRegionsDecorations.get(e.document.uri.toString());
+            //if (valuePair !== undefined) {
```

Comment:
Why is this commented out?

Reply:
it should be uncommented.

Reply:
@bobbrow is correct. Apologies while I get used to the code review cycle. All tips and advice and nitpicks are welcome. :smile: